projects/table-builder/src/lib/directives/multi-sort.directive.ts

```diff
@@ -13,56 +14,33 @@ import { scan, filter, map } from 'rxjs/operators';
 })
 export class MultiSortDirective extends MatSort implements OnInit, OnDestroy {
   @Output() readonly multiSortChange: EventEmitter<Sort[]> = new EventEmitter<Sort[]>();
-
-  @Input() rules$: Observable<Sort[]> = new Subject();
-
-  @Input() set  reSort (val){
```

Comment:
So we don't allow for resetting the sort anymore, or does it happen somewhere else?

Reply:
There was no way to do it anyway from the UI so it's not necessary.
The better way to do it from this point on would be to dispatch an action and not via the directive.